src/main/java/nextstep/subway/ui/StationController.java

```diff
@@ -1,15 +1,24 @@
 package nextstep.subway.ui;
 
+import static nextstep.subway.ui.UrlConstant.CREATE_STATIONS;
+import static nextstep.subway.ui.UrlConstant.DELETE_STATIONS;
+import static nextstep.subway.ui.UrlConstant.SHOW_STATIONS;
+
+import java.net.URI;
+import java.util.List;
 import nextstep.subway.application.StationService;
 import nextstep.subway.dto.StationRequest;
 import nextstep.subway.dto.StationResponse;
 import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
-
-import java.net.URI;
-import java.util.List;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
 
 @RestController
```

Comment:
controller에서 사용될 url prefix로 상수 값을 사용하는 것 보다는,
`@RestController`의 `value` 값을 설정하는게 좋을 것 같아요.
```suggestion
@RestController("/stations")
```

Reply:
일단 url prefix 상수 값은 전부 제거 했습니다. 

음 혹시 url 중복 때문인가요? `@RequestMapping` 으로 선언하라는 말씀이신가요?
`@RestController` 의 `value` 값은 bean 이름 인거 같은데요. 제가 이해가 잘 안되서 어떤 이유인지 알 수 있을까요?

Reply:
네, 말씀해주신 것처럼 `@RequestMapping`이 맞습니다!
제가 오타를 쳤네요 🥲